### PR TITLE
Additional EPL 2.0 license updates.

### PR DESCRIPTION
**Pull request by jhanders34**
#build

### CODE REVIEW

File: dev/io.openliberty.security.jakartasec.3.0.internal_fat.common/build.gradle [hunk 1]
```diff
@@ -2,7 +2,7 @@
  * Copyright (c) 2022 IBM Corporation and others.
  * All rights reserved. This program and the accompanying materials
  * are made available unde:0
```
Comment (cbridgha) [inline dev/io.openliberty.security.jakartasec.3.0.internal_fat.common/build.gradle]: This doesn't look right

Comment (jhanders34) [reply dev/io.openliberty.security.jakartasec.3.0.internal_fat.common/build.gradle]: Fixed

**Review by cbridgha — DISMISSED** [review]
Changes look good

**Comment by jhanders34** [comment]
#libby

**Comment by jhanders34** [comment]
#libby #build

**Comment by dmuelle** [comment]
looking at the libby message, I dont think there are any en files here that need ID review. Adding the ID review not needed label, let me know if that's incorrect.

**Review by cbridgha — APPROVED** [review]
approved

**Comment by jhanders34** [comment]
#libby

**Pull request merged**